packages/py-moose-lib/moose_lib/dmv2/olap_dictionary.py

```
@@ -0,0 +1,557 @@
+        f"dictGet('local.dict_products', 'product_name', product_id)"
+        # or via helper:
+        product_dict.get("product_name", "product_id")
+
+    Args:
+        name: Dictionary name (used in ClickHouse DDL).
+        config: OlapDictionaryConfig with source, layout, lifetime, etc.
+    """
+
+    kind: str = "OlapDictionary"
+
+    def __init__(self, name: str, config: OlapDictionaryConfig, **kwargs):
```

Ruff (0.15.7)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)